source/ore/guidelines.rst

```diff
@@ -20,8 +20,22 @@ Name
 ~~~~
 
 Your project’s submitted name should not include a version, tagline, or other description. The name should be unique and original
-and must not have a name implying it is a Sponge project (e.g. *SpongeWarp* is not allowed, *Cool Warps for Sponge* is).
-
+and must not have a name implying it is an official Sponge project.
+
+Examples of names that are **wrong**:
+- *SpongeWarp*
```

Comment:
You have to add an empty line before the first entry to make this bulleted list work.

````text
text:

* entry 1
* entry 2
````